agent/grpc-external/services/peerstream/stream_resources.go

```diff
@@ -474,8 +474,8 @@ func getTrustDomain(store StateStore, logger hclog.Logger) (string, error) {
 	return connect.SpiffeIDSigningForCluster(cfg.ClusterID).Host(), nil
 }
 
-func (s *Server) StreamStatus(peer string) (resp Status, found bool) {
-	return s.Tracker.StreamStatus(peer)
+func (s *Server) StreamStatus(peerID string) (resp Status, found bool) {
```

Comment:
Even just as a passthrough, this confused me if it wanted the name or ID for the peering so I renamed it to make it clearer to myself